guacamole-common-js/src/main/webapp/modules/Client.js

```diff
@@ -42,7 +42,36 @@ Guacamole.Client = function(tunnel) {
     var currentState = STATE_IDLE;
     
     var currentTimestamp = 0;
-    var pingInterval = null;
+
+    /**
+     * The rough number of milliseconds to wait between sending keep-alive
+     * pings. This may vary depending on how frequently the browser allows
+     * timers to run, as well as how frequently the client receives messages
+     * from the server.
+     *
+     * @private
+     * @constant
+     * @type {!number}
+     */
+    var KEEP_ALIVE_FREQUENCY = 5000;
```

Comment:
We use the `const` keyword pretty extensively in the webapp, though not yet in `guacamole-common-js`. Do you think it'd be appropriate to use that here?

Reply:
Yes, except that the guacamole-common-js build does not currently leverage the same sort of JavaScript compatibility compilation. We're safe to use new fanciness like `const` and `let` within the webapp, but guacamole-common-js would need to migrate from `minify-maven-plugin` to `frontend-maven-plugin` to get those build tools and associated benefits.

Reply:
Well, never mind for now.